### PR TITLE
gitlab: improve documentation

### PR DESCRIPTION
**Pull request by catenacyber**
cc @jonathanmetzman

### CODE REVIEW

**Comment by catenacyber** [comment]
cf https://github.com/google/clusterfuzzlite/issues/100

**Review by jonathanmetzman — COMMENTED** [review]
Mostly LGTM, just one Q

**Comment by jonathanmetzman** [comment]
Thanks!

File: docs/running-clusterfuzzlite/gitlab.md [hunk 1]
```diff
@@ -96,7 +99,7 @@ clusterfuzzlite-corpus:
   image:
     name: gcr.io/oss-fuzz-base/clusterfuzzlite-run-fuzzers:v1
     entrypoint: [""]
-  stage: fuzz
+  stage: test
```
[the remark this answers is not in the document]
Comment (catenacyber) [reply docs/running-clusterfuzzlite/gitlab.md:102]: cf default pipeline stages in https://docs.gitlab.com/ee/ci/yaml/#stages

To keep `stage: fuzz`, 
you need to add `stages:  - fuzz` somewhere (with an s)

**Pull request merged**